cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
+		o := currentVersion()
+		fmt.Println(o)
+
+		if checkLatest {
+			c := cli.New(Version)
+			l, err := checkForUpdate(c)
+			if err != nil {
+				Bail(err)
+			}
+			fmt.Println(l)
+		}
+
+		return
 	},
 }
 
+// currentVersion returns a formatted version string for the Exercism CLI.
+func currentVersion() string {
+	cmdName := RootCmd.Use
+	return fmt.Sprintf("%s version %s", cmdName, Version)
+}
+
+// checkForUpdate returns a string indicating if the current running version of the Exercism CLI is at the latest release.
+// For CLI versions found to be out of date a string with upgrade instructions in included in the return.
+func checkForUpdate(c *cli.CLI) (o string, err error) {
+
+	ok, err := c.IsUpToDate()
+	if err != nil {
+		return "", err
+	}
+
+	o = "Your CLI version is up to date."
```

Comment:
What do you think of removing the named returns, and changing this to `o := `. I feel like named returns should be there to add a descriptive context for what is being returned.